requirements/constraints.txt

```diff
@@ -14,11 +14,6 @@ futures ; python_version == "2.7"
 python-openid ; python_version == "2.7"
 pysqlite ; python_version == "2.7"
 
-# pip-tools 3.9.0 has a few problems:
-#   https://github.com/jazzband/pip-tools/issues/851
-#   https://github.com/jazzband/pip-tools/issues/852
```

Comment:
This second issue is still open.  Do we know why it isn't affecting us?

Reply:
I see that we are getting the `[django]` extra we need on opaque-keys. maybe that issue should be closed?

Reply:
Not sure,  I couldn't find anything in their changelog here: https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md

But I also couldn't re-produce it for us.

Reply:
According to the reply to your bug report, it's an intermittent problem that affects even the version we've been pinned to; we just pull in those dependencies anyway from other dependency lines, so it hasn't been impacting us.  https://github.com/jazzband/pip-tools/issues/852